features/step_definitions/logging.rb

```diff
@@ -359,3 +359,22 @@
     step %Q/logging operators are installed successfully/
   }
 end
+
+Given /^logging eventrouter is installed in the cluster$/ do
+  step %Q/I switch to cluster admin pseudo user/
+  step %Q/I use the "openshift-logging" project/
```

Comment:
Duplicate steps
```
    Given I switch to cluster admin pseudo user
    Given I use the "openshift-logging" project
    Given logging eventrouter is installed in the cluster
```

Reply:
I'll remove the steps  in the feature file
```Given I switch to cluster admin pseudo user
    Given I use the "openshift-logging" project```